### PR TITLE
Ethereum Client Library & HCL config file [sc-1745]

### PR DESCRIPTION
**Pull request by MDobak**

### CODE REVIEW

File: config.hcl
Comment (teghnet) [inline config.hcl:1]: I am more convinced now that we can drop the `CFG_` prefix

**Comment by teghnet** [comment]
Definitely SQUASH

**Comment by MDobak** [comment]
@teghnet We always do squash merges

File: config.hcl [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+  "0xA8EB82456ed9bAE55841529888cDE9152468635A",
+  "0x130431b4560Cd1d74A990AE86C337a33171FF3c6",
+  "0x8aFBD9c3D794eD8DF903b3468f4c4Ea85be953FB",
+  "0xd94BBe83b4a68940839cD151478852d16B3eF891",
+  "0xC9508E9E3Ccf319F5333A5B8c825418ABeC688BA",
+  "0x77EB6CF8d732fe4D92c427fCdd83142DB3B742f7",
+  "0x3CB645a8f10Fb7B0721eaBaE958F77a878441Cb9",
+  "0x4f95d9B4D842B2E2B1d1AC3f2Cf548B93Fd77c67",
+  "0xaC8519b3495d8A3E3E44c041521cF7aC3f8F63B3",
+  "0xd72BA9402E9f3Ff01959D6c841DDD13615FFff42"
```
[the remark this answers is not in the document]
Comment (teghnet) [reply config.hcl:28]: true, fixed

**Review by teghnet — CHANGES_REQUESTED** [review]
Maybe a good idea would be to move the extended comments from README to the config.hcl - closer to where it is actually useful.

also: https://github.com/chronicleprotocol/oracle-suite/pull/132/files#r1150663460

**Review by teghnet — APPROVED** [review]
Logically and structurally I am ok with it.

**Review by konstantinzolotarev — APPROVED** [review]

**Pull request merged**